rebar.config

```diff
@@ -25,6 +25,7 @@
 {deps, [
         {erlcloud,    ".*", {git, "https://github.com/gleber/erlcloud.git", {tag, "2.2.16"}}},
         {getopt,      ".*", {git, "https://github.com/jcomellas/getopt.git",{branch, "master"}}},
+        {libleofs,    ".*", {git, "https://github.com/mocchira/libleofs.git",{branch, "feature/add_all_commands_1.4.1"}}},
```

Comment:
@mocchira I've found a conflict of jsx's version of  between `erlcloud` and `libleofs`:

### Error Log:
```
==> libleofs (compile)
WARN:  /home/yosuke/dev/projects/leo-project/libs/leofs_test2/deps/jsx/src/jsx.app.src has version "2.8.0"; requested regex was 2.9.0
WARN:  /home/yosuke/dev/projects/leo-project/libs/leofs_test2/deps/jsx/src/jsx.app.src has version "2.8.0"; requested regex was 2.9.0
Dependency not available: jsx-2.9.0 (undefined)
ERROR: compile failed while processing /home/yosuke/dev/projects/leo-project/libs/leofs_test2/deps/libleofs: rebar_abort
Makefile:12: recipe for target 'compile' failed
make: *** [compile] Error 1
```

### Cause:

```
$ grep jsx deps/*/rebar.config
deps/erlcloud/rebar.config:        {jsx, "2.8.0"},
deps/libleofs/rebar.config:  {jsx, "~>2.9.0"}
```

Reply:
@yosukehara Thanks for the report. I've come to the conclusion how to fix this problem (version conflict) and also working with packages managed by rebar3 like the upstream libleofs without any forks. I'll update PR later.